### PR TITLE
add aside reminding users they can't script chrome://extensions

### LINKED ISSUE #1018: [extensions][getstarted] - green button in final code is not 'clickable' / does nothing
Go to https://developer.chrome.com/docs/extensions/mv3/getstarted/
Follow tutorial or download zip
Open new tab to run extension
Green button can not be clicked/does nothing

OS Win10
Chrome Version 91.0.4472.164

### PR DESCRIPTION
**Pull request by samthor**
Fixes #1018

### CODE REVIEW

File: site/en/docs/extensions/mv2/getstarted/index.md [hunk 1]
```diff
@@ -328,6 +328,11 @@ The extension is now fully functional! Reload the extension, refresh this page,
 click the button to turn it green! However, some users may want to change the background to a
 different color.
 
+{% Aside 'gotchas' %}
+You can't script any internal Chrome page, like "chrome://extensions". Be sure to try out the
```
Comment (awfuchs) [inline site/en/docs/extensions/mv2/getstarted/index.md:332]: It isn't obvious what "script a page" means. Can you rephrase?

File: site/en/docs/extensions/mv3/getstarted/index.md [hunk 2]
```diff
@@ -327,6 +327,11 @@ The extension is now fully functional! Reload the extension, refresh this page,
 click the button to turn it green! However, some users may want to change the background to a
 different color.
 
+{% Aside 'gotchas' %}
+You can't script any internal Chrome page, like "chrome://extensions". Be sure to try out the
```
Comment (awfuchs) [inline site/en/docs/extensions/mv3/getstarted/index.md:331]: Ditto.

**Review by awfuchs — CHANGES_REQUESTED** [review]

File: site/en/docs/extensions/mv3/getstarted/index.md (hunk 2 above)
Comment (samthor) [reply site/en/docs/extensions/mv3/getstarted/index.md]: Changed to "You can't run a content script on an internal Chrome page, such as ...".

File: site/en/docs/extensions/mv2/getstarted/index.md [hunk 3]
```diff
@@ -328,6 +328,11 @@ The extension is now fully functional! Reload the extension, refresh this page,
 click the button to turn it green! However, some users may want to change the background to a
 different color.
 
+{% Aside 'gotchas' %}
+You can't run a content script on an internal Chrome page, such as "chrome://extensions". Be sure
+to try out the extension on a real webpage, like https://google.com.
```
Comment (dotproto) [inline site/en/docs/extensions/mv2/getstarted/index.md:333]:
```suggestion
Extensions can not inject content scripts on internal Chrome pages like "chrome://extensions". Be
sure to try out the extension on a real webpage like [https://google.com](https://google.com).
```

Rephrase slightly to emphasize that this is a platform restriction, not an issue with the reader.

Comment (dotproto) [inline site/en/docs/extensions/mv2/getstarted/index.md:333]:
```suggestion
Extensions can not inject content scripts on internal Chrome pages like "chrome://extensions". Be
sure to try out the extension on a real webpage like [https://google.com](https://google.com).
```

Rephrase slightly to emphasise that this is a platform restriction, not an issue with the reader.

File: site/en/docs/extensions/mv3/getstarted/index.md [hunk 4]
```diff
@@ -327,6 +327,11 @@ The extension is now fully functional! Reload the extension, refresh this page,
 click the button to turn it green! However, some users may want to change the background to a
 different color.
 
+{% Aside 'gotchas' %}
+You can't run a content script on an internal Chrome page, such as "chrome://extensions". Be sure
+to try out the extension on a real webpage, like https://google.com.
```
Comment (dotproto) [inline site/en/docs/extensions/mv3/getstarted/index.md:332]:
```suggestion
Extensions can not inject content scripts on internal Chrome pages like "chrome://extensions". Be
sure to try out the extension on a real webpage like [https://google.com](https://google.com).
```

**Review by dotproto — APPROVED** [review]
Small nit with the current phrasing, but other than that this is a nice improvement. Thanks, @samthor.
